MentalHealth/Resources.csv

```diff
@@ -0,0 +1,15 @@
+Title,Description,Location,Hours,Email,Phone,Phoneroot,Website,Tags
```

Comment:
@Ankurmangal For consistency with our other datasets, can you make these titles lower case please.